straxen/analyses/posrec_comparison.py

```diff
@@ -13,7 +13,7 @@ def load_corrected_positions(
     alt_s1=False,
     alt_s2=False,
     cmt_version=None,
-    posrec_algos=("mlp", "gcn", "cnn"),
```

Comment:
Why no `"cnf"` here?

Reply:
It compares the FDC map, and the `cnf` one is currently unavailable

Reply:
Please make an issue about this.